Assets/UI/Choosers/ResearchChooser_CQUI_Basegame.lua

```diff
@@ -0,0 +1,7 @@
+-- ===========================================================================
+-- Base File
+-- ===========================================================================
+-- Note: There is no ReseachChooser_Expansion2, as ResearchChooser_Expansion1 (one) is the file that is bundled with both the Rise and Fall and Gathering Storm expansions
+include("ResearchChooser");
```

Comment:
Base file is loaded twice: 1st time here and 2nd time in ResearchChooser_CQUI.lua.